prbt_hardware_support/src/brake_test_executor_node.cpp

```diff
@@ -34,6 +35,7 @@ static const std::string CONTROLLER_HOLD_MODE_SERVICE_NAME{"/prbt/manipulator_jo
 static const std::string CONTROLLER_UNHOLD_MODE_SERVICE_NAME{"/prbt/manipulator_joint_trajectory_controller/unhold"};
 static const std::string BRAKE_TEST_RESULT_SERVICE_NAME{"/prbt/send_brake_test_result"};
 
+using namespace pilz_utils;
```

Comment:
I would be in favor of addressing the function directly with namespace or at least defining an alias if you consider "pilz_utils" to be to long.
I hope that generally we did not "using" on other packages that often. I like the fact that this hints that another package is used because especially if this package lives in another repo some day this could be useful

Reply:
I changed it.

Reply:
42e6b16d3497e82f8a976b0c5691ac5a3eb9cd89